packages/graphiql/src/components/GraphiQL.tsx

```diff
@@ -958,8 +964,32 @@ export class GraphiQL extends React.Component<GraphiQLProps, GraphiQLState> {
       });
 
       return subscription;
+    } else if (isAsyncIterable(fetch)) {
+      (async () => {
+        try {
+          for await (const result of fetch) {
```

Comment:
https://caniuse.com/mdn-javascript_statements_for_await_of

Reply:
Should be all fine given that we don't support IE11 :)

Reply:
all of this will be passed through browserslist and babel anyways

also to note re: `for await of`: afaik this will gaurantee serial execution, this is desired?